shardy/dialect/sdy/ir/ops.td

```diff
@@ -137,6 +137,15 @@ def Sdy_ManualComputationOp : Sdy_Op<"manual_computation",
 
     The body is local wrt the manual_axes. Propagation will occur through
     the body on any free axes - those not in the manual_axes list.
+
+    Constraints:
+
+      For each set of op values (operands/results) and corresponding sharding for each value:
+      - The in/out shardings must valid w.r.t the corresponding global type
+      - The number of global and local tensor inputs/outputs of the op region must match
+      - The manual axes must come before any free axes in each dim sharding
+      - The global shape and local shapes of the op regions arguments/results must match
+      - No manual axes must be split.
```

Comment:
hyper nit: "No manual axes are split"

Reply:
I kept wondering as a non-native speaker what would be the best wording here 😄